lib/flows/login-flow.js

```diff
@@ -65,6 +65,9 @@ export default class LoginFlow {
 		}
 
 		loginPage = new LoginPage( this.driver, true );
+		this.driver.getCurrentUrl().then( ( urlDisplayed ) => {
+			return loginPage.setABTestControlGroupsInLocalStorage( urlDisplayed );
```

Comment:
This doesn't work because the (older) login page is on a separate domain (wordpress.com not wpcalypso.wordpress.com) - and the ab test keys in localstorage won't be set for wpcalypso.wordpress.com